examples/centrality-algorithms.ipynb

```diff
@@ -438,13 +462,29 @@
     "- For the city metadata (metropolitan population, latitude, and longitude):\n",
     "Austin R. Benson, David F. Gleich, and Jure Leskovec. \"Higher-order Organization of Complex Networks.\" Science, 353.6295 (2016): 163–166.\n",
     "\n",
+    "- Link to the dataset: https://snap.stanford.edu/data/reachability.html\n",
+    "\n",
     "- Notebook contributed by [Kedar Ghule](https://github.com/kedarghule)"
    ]
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
   "language_info": {
-   "name": "python"
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.11"
```

Comment:
This stuff was added back in, which is normal when the notebook is run. I will clear it out again.